pkg/kubelet/rkt/rkt.go

```diff
@@ -51,8 +51,8 @@ import (
 )
 
 const (
-	acVersion             = "0.6.1"
-	rktMinimumVersion     = "0.8.1"
+	acVersion             = "0.7.1"
+	supportedRktVersion   = "0.9.0"
```

Comment:
:+1: for relaxing dependency version (even with ["semver"](http://semver.org/) scheme doesn't guarantee anything until '1.0.0')